src/darwin/Framework/CHIP/CHIPDeviceController.mm

```diff
@@ -38,14 +38,14 @@
 static NSString * const kErrorUnpairDevice = @"Failure while unpairing the device";
 static NSString * const kErrorStopPairing = @"Failure while trying to stop the pairing process";
 static NSString * const kErrorGetPairedDevice = @"Failure while trying to retrieve a paired device";
+static NSString * const kInfoStackShutdown = @"Shutting down the CHIP Stack";
 
 @interface CHIPDeviceController ()
 
 @property (nonatomic, readonly, strong, nonnull) NSRecursiveLock * lock;
 
-// queue used to call select on the system and inet layer fds., remove this with NW Framework.
-// primarily used to not block the work queue
-@property (atomic, readonly) dispatch_queue_t chipSelectQueue;
+// queue used to serialize access to the underlying cppCommissioner
```

Comment:
It's also used for access to `_persistentStorageDelegateBridge`, right?  In general, we should ideally document next to the variables that need to be accessed via the queue that they need to be thus accessed...  Followup on this, please?

Reply:
Yeah, actually this is used to serialize all work performed by the DeviceController. Not just variable access. I'll update the comment.